### PR TITLE
Updated development ruby version

### PR DESCRIPTION
**Pull request by miry**
Added current actual ruby versions.
Removed custom safe_yml, it is not required anymore.

### CODE REVIEW

**Comment by tiagotex** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,13 @@
 language: ruby
 script: bundle exec rspec spec
+before_install:
```
Comment (antlypls) [inline .travis.yml]: Why do we need this?

**Comment by antlypls** [comment]
And btw, why are there two CIs (travis and circle) configured for the project?

**Comment by miry** [comment]
Travis allow us to run tests for different ruby versions.
I suppose CircleCi was added for fast response.

File: .travis.yml (hunk 1 above)
Comment (miry) [inline .travis.yml]: there is buggy bundler version in travis for some rubies.

**Pull request merged**